modules/site/src/app/examples/pagination/pagination-basic-usage.component.spec.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  beforeEach,
+  beforeEachProviders,
+  describe,
+  expect,
+  it,
+  inject,
+} from '@angular/core/testing';
+import { ComponentFixture, TestComponentBuilder } from '@angular/compiler/testing';
+import { Component } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { PaginationBasicUsageComponent } from './pagination-basic-usage.component';
+
+describe('Component: PaginationBasicUsage', () => {
+  let builder: TestComponentBuilder;
+
+  beforeEachProviders(() => [PaginationBasicUsageComponent]);
+  beforeEach(inject([TestComponentBuilder], function (tcb: TestComponentBuilder) {
+    builder = tcb;
+  }));
+
+  it('should inject the component', inject([PaginationBasicUsageComponent],
```

Comment:
It can be useful to have these tests, because they do a quick sanity check that the component can be initialized and destroyed at runtime, but I think we should DRY it up. Also, for it to be useful I think we would need to run these tests as part of the build process.

We could do two things:

 - drop the tests from the example project entirely (and investigate how to create components without them)
 - DRY up the example tests using a helper function like [componentSanityCheck](https://github.com/justindujardin/ng2-material/blob/master/src/platform/testing/util.ts#L19), update the contributing docs, and update the travis script to run the site tests.

I'm open to both options.